packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -334,6 +340,7 @@ function BackgroundImageControls( {
 					! positionValue && ( 'auto' === sizeValue || ! sizeValue )
 						? '50% 0'
 						: positionValue,
+				backgroundSize: sizeValue,
```

Comment:
Uploaded images will receive any incoming default values. 

For example, `hooks/background.js` defines `"cover"` - this will affect blocks only.